manual_flagging/requirements.txt

```diff
@@ -9,12 +9,12 @@ idna==3.4
 iniconfig==2.0.0
 jmespath==1.0.1
 joblib==1.3.2
-numpy==1.24.3
+numpy==1.23.4
 packaging==23.1
-pandas==2.0.2
+pandas==1.3.5
 pluggy==1.2.0
-pyarrow==12.0.1
-pyathena==3.0.6
+pyarrow==10.0.0
+pyathena==2.25.2
```

Comment:
I made the corresponding package changes to the venv in local so we can maintain a single source of truth for the flagging script, and so we can have a more consistent environment between local and glue. Changes were made to `pyarrow` and `numpy` to be compatible with the earlier package versions from the spark glue job.